libs/blocks/merch-card/merch-card.css

```diff
@@ -0,0 +1,376 @@
+}
+
+.merch-card .consonant-PlansCard-title h2 {
+  font-size: 24px;
+  line-height: 30px;
+}
+
+.merch-card .consonant-SpecialOffers-title h3,
+.merch-card .consonant-PlansCard-title h3 {
+  font-size: 18px;
+  font-weight: 700;
+  margin-bottom: 0;
+}
+
+.merch-card .consonant-PlansCard-title h3 {
+  font-size: 18px;
+  font-weight: 700;
+  margin-bottom: 8px;
+  line-height: 22.5px;
+}
+
+.merch-card .consonant-SpecialOffers-title h4,
+.merch-card .consonant-PlansCard-title h4 {
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.merch-card .consonant-SpecialOffers-title h5,
+.merch-card .consonant-PlansCard-title h5,
+.merch-card .consonant-SegmentBlade-title h5 {
+  font-size: 12px;
+  color: #2C2C2C;
+  font-weight: 500;
+}
+
+.merch-card .consonant-ProductCard-prices,
+.merch-card .consonant-SegmentBlade-prices,
+.merch-card .consonant-SpecialOffers-prices,
+.merch-card .consonant-PlansCard-prices {
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 8px;
+  font-weight: 700;
+}
+
+.merch-card .consonant-ProductCard-description,
+.merch-card .consonant-SegmentBlade-description,
+.merch-card .consonant-SpecialOffers-description,
+.merch-card .consonant-PlansCard-description {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-top: 0;
+}
+.merch-card .consonant-PlansCard-description {
+  margin-bottom: 18px
+}
+
+.merch-card .consonant-SegmentBlade-description span.placeholder-resolved,
+.merch-card .consonant-SpecialOffers-description span.placeholder-resolved,
+.merch-card .consonant-PlansCard-description span.placeholder-resolved {
+  font-size: 18px;
+  font-weight: 700;
+}
+
+.merch-card .consonant-SegmentBlade-description span.placeholder-resolved[data-template="priceStrikethrough"],
+.merch-card .consonant-SpecialOffers-description span.placeholder-resolved[data-template="priceStrikethrough"],
+.merch-card .consonant-PlansCard-description span.placeholder-resolved[data-template="priceStrikethrough"] {
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.merch-card.consonant-SpecialOffers-Card {
+  border-radius: 16px;
+}
+
+.merch-card .consonant-SpecialOffers-img,
+.merch-card .consonant-PlansCard-img {
+  flex-grow: 1;
+  position: relative;
+  width: 100%;
+  min-height: 213px;
+  max-height: 213px;
+  background-color: #eaeaea;
+  background-position: 50% 50%;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.merch-card .consonant-SpecialOffers-ribbon,
+.merch-card .consonant-SegmentBlade-ribbon,
+.merch-card .consonant-PlansCard-ribbon {
+  position: absolute;
+  top: 16px;
+  right: 0;
+  font-size: 14px;
+  font-weight: 500;
+  max-width: 150px;
+  line-height: 16px;
+  text-align: center;
+  padding: 8px 11px 8px 11px;
+  border-radius: 5px 0 0 5px;
+}
+
+.merch-card .consonant-ProductCard-cell,
+.merch-card .consonant-SegmentBlade-cell,
+.merch-card .consonant-SpecialOffers-cell,
+.merch-card .consonant-PlansCard-cell {
+  justify-content: flex-end;
+  align-items: center;
+}
+
+p {
```

Comment:
see comment below. Too wide open selector and we should use typography.css classes where possible

Reply:
Is there a resolve for this text/button overflow overlap?
<img width="1048" alt="Screen Shot 2023-08-15 at 1 36 21 PM" src="https://github.com/adobecom/milo/assets/2086146/00c6f7de-be6e-4938-9ddc-2fe2e788c5d6">

Reply:
done.